Component versions (#86)
mulesoft/docs-private-cloud: modules/ROOT/pages/upgrade.adoc

+
By performing these procedures from the master node, the upgrade scripts are able to update the entire platform.

> . Verify there is adequate space available in the path where you plan to download the new installer file.

Reviewer: ```suggestion
. Verify that there is adequate space available in the path where you plan to download the new installer file.
```

----
-> initializing upload
-> uploading update package
importing anypoint:2.1.0
Thu Aug 23 09:36:30 UTC	Synchronizing application with Docker registry x.x.x.xxx:5000
Thu Aug 23 09:44:27 UTC	Synchronizing application with Docker registry x.x.x.xxx:5000
Thu Aug 23 09:51:29 UTC	Synchronizing application with Docker registry x.x.x.xxx:5000
Thu Aug 23 09:58:23 UTC	Application has been uploaded
updating anypoint from 2.0.4 to 2.1.0-xxxxxxxxxxxx
update operation (d077277f-3c69-4336-82bf-bd8b428ae202) has been started
the cluster is updating in background
----

> . When the upgrade finishes, verify the migration was successful by checking that all pods are running:

Reviewer: ```suggestion
. When the upgrade finishes, verify that the migration was successful by checking that all pods are running:
```

+
----
$ sudo gravity enter
$ kubectl get pods --all-namespaces
----

. Open the new `8083` port on your load balancer and map it to the `30883` port on your servers. Refer to the 2.1 xref:prereq-hardware.adoc[Hardware Prerequisities] section for more information. 

> . Finally, make sure the following items are still correctly configured after the update:

Reviewer: I know these seem trivial, but I am thinking of a global audience, which can have trouble with colocations (see style ref):
```suggestion
. Finally, ensure that the following items are still correctly configured after the update:
```
